copier/main.py

```diff
@@ -865,6 +902,22 @@ def run_auto(
     return run_copy(src_path, dst_path, data, **kwargs)
 
 
+def run_check(
+    dst_path: StrOrPath = ".",
+    data: Optional[AnyByStrDict] = None,
+    **kwargs,
+) -> Worker:
+    """Check if a subproject is using the latest versio of its template.
+
+    See [Worker][copier.main.Worker] fields to understand this function's args.
+    """
+    if data is not None:
+        kwargs["data"] = data
```

Comment:
The `check` command doesn't use data anyway, so I think this block can be removed.

```suggestion
```